tuna/export_db.py

```diff
@@ -57,6 +57,8 @@ def parse_args():
                       action='store_true',
                       help='Use OpenCL extension',
                       default=False)
+  # pylint: disable-all
+  # pylint false flag this as duplicate with line 134 in go_fish file
```

Comment:
since we are using session_id in many scripts now, this can probably be moved to parse_args.py and imported in all the scripts that use it just like we do with arch, num_cu, config_type etc.